src/app/navbar/navbar.component.ts

```diff
@@ -58,13 +68,42 @@ export class NavbarComponent extends Logout implements OnInit {
       .subscribe(() => {
         this.isExtended = toExtendSite(this.router.url);
       });
+    this.isAdminOrCurator$ = this.userQuery.isAdminOrCurator$;
   }
 
   ngOnInit() {
     this.userQuery.user$.pipe(takeUntil(this.ngUnsubscribe)).subscribe((user) => {
       this.user = user;
+      this.requestsService.updateMyMemberships();
+      this.requestsService.updateCuratorOrganizations();
     });
     this.userQuery.extendedUserData$.pipe(takeUntil(this.ngUnsubscribe)).subscribe((extendedUser) => (this.extendedUser = extendedUser));
+    this.myOrganizationInvites$ = this.requestsQuery.myOrganizationInvites$;
+    this.myRejectedOrganizationRequests$ = this.requestsQuery.myRejectedOrganizationRequests$;
+    this.allPendingOrganizations$ = this.requestsQuery.allPendingOrganizations$;
+    this.notificationCount$ = combineLatest([
+      this.myOrganizationInvites$,
+      this.myRejectedOrganizationRequests$,
+      this.allPendingOrganizations$,
+      this.isAdminOrCurator$,
+    ]).pipe(
+      takeUntil(this.ngUnsubscribe),
+      map(
+        ([invites, rejections, pendingOrganizations, isAdminOrCurator]: [
+          Array<OrganizationUser>,
+          Array<OrganizationUser>,
+          Array<Organization>,
+          boolean
+        ]) => {
+          let count = invites?.length + rejections?.length;
```

Comment:
I don't think this will always work. Note that if you use `?` and if the value doesn't exist, you get `undefined`. So suppose `invites?.length` resolves to 5 and `rejections?.length` resolves to `undefined`, then `5 + undefined` equals `undefined`, but you would want it to be 5.

In my other comment, I suggested using the TypeScript [nullish coalescing operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing), and I still do. Then you don't need any isNaN checks, you know it will be a number. Example:

```
let count = (invites?.length ?? 0) + (rejections?.length ?? 0);
```

Reply:
changed!